deploy/clowdapp.yaml

```diff
@@ -200,3 +202,7 @@ parameters:
   name: SOURCES_ENV
   required: true
   value: stage
+- description: Host for the EAN to OrgId translator
+  name: TENANT_TRANSLATOR_HOST
+  required: true
+  value: '?????????'
```

Comment:
Do you know where I can find the tenant translator host for this one? I tried finding it on the documents to give the variable a sane default but... :sweat: